agent/eni/networkutils/utils_linux.go

```diff
@@ -43,23 +134,23 @@ func IsValidNetworkDevice(devicePath string) bool {
 	* eth1 -> /devices/pci0000:00/0000:00:05.0/net/eth1
 	* eth0 -> ../../devices/pci0000:00/0000:00:03.0/net/eth0
 	* lo   -> ../../devices/virtual/net/lo
-	*/
+	 */
 	splitDevLink := strings.SplitN(devicePath, "devices/", 2)
 	if len(splitDevLink) != 2 {
-		log.Warnf("Cannot determine device validity: %s", devicePath)
+		seelog.Warnf("Cannot determine device validity: %s", devicePath)
```

Comment:
its not clear to me why `len(splitDevLink) != 2` means we cannot determine device validity. can we add some documentation? should we add the `splitDevLink` to log output as well? although, maybe i'm just missing context.

Reply:
The example entries in the comment above are meant to illustrate that:

`eth1 -> /devices/pci0000:00/0000:00:05.0/net/eth1`

This means that we expect it to be of "devices/some-string" format.